docs/products/kafka/kafka-connect/howto/bring-your-own-kafka-connect-cluster.rst

```diff
@@ -0,0 +1,206 @@
+
+The following article defines the necessary steps to integrate your own Apache Kafka Connect cluster with Aiven for Apache Kafka and use the schema registry offered by `Karapace <https://help.aiven.io/en/articles/5651983>`__. The example shows how to create a JDBC sink connector to a PostgreSQL® database.
+
+.. _bring_your_own_kafka_connect_prereq:
+
+Prerequisites
+-------------
+
+To bring your own Apache Kafka Connector, you need an Aiven for Apache Kafka service up and running. 
+
+Furthermore, for the JDBC sink connector database example, you need to collect the following information about the Aiven for Apache Kafka service and the target database upfront:
+
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service
+* ``APACHE_KAFKA_PORT``: The port of the Apache Kafka service
+* ``REST_API_PORT``: The Apache Kafka's REST API port, only needed when testing data flow with REST APIs
+* ``REST_API_USERNAME``: The Apache Kafka's REST API username, only needed when testing data flow with REST APIs
+* ``REST_API_PASSWORD``: The Apache Kafka's REST API password, only needed when testing data flow with REST APIs
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+* ``PG_HOST``: The PostgreSQL service hostname
+* ``PG_PORT``: The PostgreSQL service port
+* ``PG_USERNAME``: The PostgreSQL service username
+* ``PG_PASSWORD``: The PostgreSQL service password
+* ``PG_DATABASE_NAME``: The PostgreSQL service database name
+
+.. Note::
+
+    If you're using Aiven for PostgreSQL and Aiven for Apache Kafka the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+
+Attach your own Apache Kafka Connect cluster to Aiven for Apache Kafka®
+-----------------------------------------------------------------------
+
+The following example demonstrates how to setup a local Apache Kafka Connect cluster with a working JDBC sink connector and attach it to an Aiven for Apache Kafka service.
+
+.. _setup_trustore_keystore_bring_your_own_connect:
+
+Setup the truststore and keystore
+''''''''''''''''''''''''''''''''''
+
+Create a :doc:`Java keystore and truststore <../../howto/keystore-truststore>` for the Aiven for Apache Kafka service.
+For the following example we assume:
+
+* The keystore is available at ``KEYSTORE_PATH/client.keystore.p12``
+* The truststore is available at ``TRUSTSTORE_PATH/client.truststore.jks``
+* The same secret (password) is used for both the keystore and the truststore, and is shown here as ``KEY_TRUST_SECRET``
```

Comment:
Thanks @TibsAtWork.
From a security standpoint is not ideal to have the same passwords, but I wouldn't loose too much time into the details of keystores since it's not the main topic of the article which is very long already. WDYT?

Reply:
I agree it's a bit late in the process to be fiddling with changes of that scope.
Maybe add a "For simplicity," in front of "the same ..." (which feels naughty, but at least hints we know we're not being good).

Reply:
For simplicity works!